mingw-w64-SDL2_gfx/PKGBUILD

```
@@ -25,11 +25,18 @@ build() {
   export lt_cv_deplibs_check_method='pass_all'
   [[ -d "${srcdir}/build-${MINGW_CHOST}" ]] && rm -rf "${srcdir}/build-${MINGW_CHOST}"
   mkdir -p "${srcdir}/build-${MINGW_CHOST}"
+
+  local extra_config=""
+  if [[ ${MINGW_CHOST} == "aarch64-w64-mingw32" ]]; then
+    extra_config+="--disable-mmx"
+  fi
```

Not a big deal, but I would do something like
```suggestion
  case "${CARCH}" in
    i686|x86_64)
      ;;
    *)
      extra_config+="--disable-mmx"
      ;;
  esac
```

---

OK, will do.